addons/website_forum/views/website_forum.xml

```diff
@@ -1531,7 +1531,7 @@
         </nav>
 
         <div class="row mb-5" t-if="tags">
-            <div class="col-md-3 mt16 o_js_forum_tag_follow" t-foreach="tags" t-as="tag">
+            <div class="col-md-3 mt16 overflow-hidden o_js_forum_tag_follow" t-foreach="tags" t-as="tag">
```

Comment:
text-truncate should be better no  ?

Reply:
Oh i have tested with both and notice i accidently cause a bug, if we add either `overflow-hidden` or `text-truncate` the popup subscribe element will not be shown and user can not see the post and subscribe. So i change to `white-space: normal` style
![image](https://github.com/odoo/odoo/assets/56789189/ceb4403b-5e62-49c1-abcb-8560f1a17641)
if you think there is another option i'll be grateful, otherwise this can be closed :((